draft-ietf-rtcweb-jsep.xml

```diff
@@ -1798,12 +1798,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 <t> "RTP/AVPF" </t>
                 <t> "RTP/SAVP" </t>
                 <t> "RTP/SAVPF" </t>
-                <t> "UDP/TLS/RTP/AVP" </t>
-                <t> "UDP/TLS/RTP/AVPF" </t>
+                <t> "TCP/DTLS/RTP/SAVP" </t>
+                <t> "TCP/DTLS/RTP/SAVPF" </t>
+                <t> "TCP/RTP/AVP" </t>
+                <t> "TCP/RTP/AVPF" </t>
+                <t> "TCP/RTP/SAVP" </t>
+                <t> "TCP/RTP/SAVPF" </t>
+                <t> "TCP/TLS/RTP/AVP" </t>
+                <t> "TCP/TLS/RTP/AVPF" </t>
                 <t> "UDP/TLS/RTP/SAVP" </t>
```

Comment:
This seems a bit inconsistent. Why do we have TCP/TLS/RTP/AVP, but not UDP/TLS/RTP/AVP?